examples/plot_line_testing.py

```diff
@@ -0,0 +1,137 @@
+# Testing the Line Plot
+# ---------------------
+# Now you can make a PlotTester object and test the line plot. You can test
+# the line type is a one to one line or a regression line, and you can test
+# that the line has the correct y intercept and slope.
+
+################################################################################
+# Testing the Line Types
+# ----------------------
+# As you can see, there are two line types on this plot. A one to one line for
+# reference, and a regression line of the points plotted. You can use the
+# function ``assert_lines_of_type()`` to test if a one to one or regression line
+# (or both types of line) are present in your plot. These are the only types
+# of lines you can currently test for with this function.
+
+# Check line types
+line_figure_tests.assert_lines_of_type(line_types=['regression', 'onetoone'])
+
+################################################################################
+# Testing the Slope and Y Intercept
+# ---------------------------------
+# Another aspect of a line plot that you can test is the slope and Y intercept,
+# which checks to ensure the line has the correct values. If you made your
+# line from a list of vertices, you can use the ``get_slope_yintercept()``
+# function in the PlotTester object in order to get the slope and y intercept.
+# However, if you made your line from a regression function, it will take an
+# extra step to get the slope and intercept data. In this example,
+# ``stats.linregress`` is used to calculate the slope and intercept data. Once
+# you have created that data, you can plug it into the ``assert_line()``
+# function to ensure your line in your plot has the correct values.
+
+# Create the slope and intercept data for the line in the plot to check against
+slope_data, intercept_data, _, _, _ = stats.linregress(
+    data.Data1, data.Data2)
+
+# Check line is correct
+line_figure_tests.assert_line(slope_exp=slope_data, intercept_exp=intercept_data)
+
+
+################################################################################
+# Access the Axes object in a Jupyter Notebook
+# --------------------------------------------
+# MatPlotCheck can be used to help grade Jupyter Notebooks as well. The main
+# difference is in how you would store the Axes from the plot you are grading.
+# Below is an example of how you could store the Axes of a plot you are hoping
+# to grade in a notebook.
+
+# First, import the Notebook module from MatPlotCheck
+import matplotcheck.notebook as nb
+
+# Plot the data
+fig, ax = plt.subplots()
+
+# Points and line of regression
+sns.regplot('Data1', 'Data2',
+            data=data,
+            color='purple',
+            ax=ax)
+
+# 1:1 line
+ax.plot((0, 1), (0, 1), transform=ax.transAxes, ls='--', c='k')
+
+ax.set(xlabel='Data1',
+       ylabel='Data2',
+       title='Example Data Regression Plot',
+       xlim=(0, 25),
+       ylim=(0, 25));
+
+# HERE'S WHERE YOU STORE THE PLOT!
```

Comment:
Change this text to match [vector vignette](https://github.com/nkorinek/matplotcheck/blob/da924a0e4374b1563120b517ae7a935fb07aa5a5/examples/plot_vector_testing.py): "Here is where you access the axes objects of the plot for testing.
You can add the code line below to the end of any plot cell to store all axes objects created by matplotlib in that cell."